layers/+window-management/exwm/packages.el

```diff
@@ -159,13 +158,7 @@
 
     (exwm/exwm-bind-command
      "s-'"  exwm-terminal-command
```

Comment:
this is duplicated.
Is this intended?

Reply:
Not sure what you're asking here – if you're referring to the `exwm-bind-command`, I think that's Github displaying the two different chunks of the diff a bit weirdly.  It only actually shows up once.
Above that is a `progn` I didn't *think* we need.  Do we need it?

Reply:
What I'm asking is there're two bindings for `exwm-terminal-command`, and
do you have both of them on purpose?

Reply:
oh I see, sorry.
This is outside of the scope of my PR, though I could remove one of them.  The `s-'` is documented in the README; `s-return` isn't.  Both make a little sense though.  `s-'` feels like a "system repl", `s-return` is more standard on other window managers.  So I could document both in the README.  Thoughts?

Reply:
let's remove s-'.
This relies on qwerty layout. On other layout ' can be anywhere on the keyboad is not necessarily easy to type.